src/error.ts

```diff
@@ -682,13 +687,15 @@ function makeWriteConcernResultObject(input: any) {
 export class MongoWriteConcernError extends MongoServerError {
   /** The result document (provided if ok: 1) */
   result?: Document;
+  errInfo?: Document;
 
-  constructor(message: ErrorDescription, result: Document) {
+  constructor(message: ErrorDescription, result?: Document) {
     if (result && Array.isArray(result.errorLabels)) {
       message.errorLabels = result.errorLabels;
     }
 
     super(message);
+    this.errInfo = message.errInfo;
```

Comment:
Related to another comment, I think I remember now this was automapped by the Object.assign in bulk right? and now its being done explicitly is that correct?

Reply:
Yup, that's correct!